### PR TITLE
/fe: added basic navigation for metrics and views

### LINKED ISSUE #32: /fe: add basic navigation for metrics and views

### PR DESCRIPTION
**Pull request by mindrones**
closes #32

### CODE REVIEW

File: fe/src/lib/components/layout/medium/ExplorerMedium.svelte [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
-	<div class='timeline' />
-	<div class='view' />
+	<nav class='filters'>Filters</nav>
+	<nav class='metrics'>
+		<MetricSelector />
+	</nav>
+	<nav class='timeline' />
+	<section class='view'>
+		<ViewMedium>
+			<slot></slot>
```
Comment (saabi) [inline fe/src/lib/components/layout/medium/ExplorerMedium.svelte]: Maybe just `<slot />`?

File: fe/src/lib/data/metrics.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {getId, objectToKeyValueArray} from '@svizzle/utils';
+import * as _ from 'lamb';
+
+import {counts, fields} from 'nesta_hpmt_shared/index.js';
+
+const getEntity = _.getKey('entity');
+
+export const all = _.sort(fields.concat(counts), [getEntity])
```
Comment (saabi) [inline fe/src/lib/data/metrics.js]: ;

File: fe/src/routes/explorer/+layout.svelte [hunk 3]
```diff
@@ -18,9 +26,11 @@
...
 <ViewsXor viewId={$_screenId}>
 	<View id='medium'>
-		<ExplorerMedium/>
+		<ExplorerMedium>
+			<slot></slot>
+		</ExplorerMedium>
 	</View>
 	<View id='small'>
-		TODO
+		<slot></slot>
```
Comment (saabi) [inline fe/src/routes/explorer/+layout.svelte]: `<slot />` and in line #30 as well

File: fe/vite.config.js [hunk 4]
```diff
@@ -2,6 +2,9 @@ import { sveltekit } from '@sveltejs/kit/vite';
 
 const config = {
 	plugins: [sveltekit()],
+	// resolve: {
+	// 	preserveSymlinks: true
+	// },
```
Comment (saabi) [inline fe/vite.config.js]: Leftover

**Review by saabi — APPROVED** [review]
Approved pending minor fixes in comments

**Review by doogyb — APPROVED** [review]

**Pull request merged**